lib/reference/functions.mjs

```diff
@@ -17,11 +17,14 @@ export const camelCaseFunctions = new Set([
 export const colorFunctions = new Set([
```

Comment:
note

This set is used by `color-named`.
https://github.com/stylelint/stylelint/blob/ebd328ada2a5574800cc83bebc5b42eaa09bbe14/lib/rules/color-named/index.mjs#L77